vtex-apps/store-components
# Fix Shipping Simulator PropType Error

#### What is the purpose of this pull request?

There was a PropType console error in the Product Page related to the *Shipping Simulator*. I fixed by changing the type from *numeric* to *string*.

#### How should this be manually tested?

[Workspace](https://elvis--storecomponents.myvtex.com/)

- Visit any product in the store
- Check if there is any Proptype errors of `seller`

#### Types of changes
